pymc3/distributions/discrete.py

```diff
@@ -807,16 +844,127 @@ def logp(self, value):
         TensorVariable
         """
         p = self.p
-        return bound(tt.log(p) + logpow(1 - p, value - 1),
-                     0 <= p, p <= 1, value >= 1)
+        return bound(tt.log(p) + logpow(1 - p, value - 1), 0 <= p, p <= 1, value >= 1)
 
     def _repr_latex_(self, name=None, dist=None):
         if dist is None:
             dist = self
         p = dist.p
-        name = r'\text{%s}' % name
-        return r'${} \sim \text{{Geometric}}(\mathit{{p}}={})$'.format(name,
-                                                get_variable_name(p))
+        name = r"\text{%s}" % name
+        return r"${} \sim \text{{Geometric}}(\mathit{{p}}={})$".format(
+            name, get_variable_name(p)
+        )
+
+
+class HyperGeometric(Discrete):
+    R"""
+    Hypergeometric log-likelihood.
+
+    The probability of x successes in a sequence of n Bernoulli
+    trials (That is, sample size = n) - where the population
+    size is N, containing a total of k successful individuals.
+    The process is carried out without replacement.
+
+    The pmf of this distribution is
+    .. math:: f(x \mid N, n, k) = \frac{\binom{k}{x}\binom{N-k}{n-x}}{\binom{N}{n}}
+    .. plot::
```

Comment:
```suggestion
    The pmf of this distribution is

    .. math:: f(x \mid N, n, k) = \frac{\binom{k}{x}\binom{N-k}{n-x}}{\binom{N}{n}}

    .. plot::
```
Nitpick. Not a blocking comment